packaging/installer/methods/manual.md

```diff
@@ -115,6 +115,16 @@ Netdata DB engine can be enabled when these are installed (they are optional):
 
 *Netdata will greatly benefit if you have the above packages installed, but it will still work without them.*
 
+Netdata Cloud support may require the following packages to be installed:
+
+| package  | description
+|:--------:| -----------------------
+| `libwebsockets-devel` } Version 3 or higher is at needed build time for websockets support for Netdata Cloud
```

Comment:
```suggestion
| `libwebsockets-devel` | Version 3 or higher is at needed build time for websockets support for Netdata Cloud
```

Reply:
Resolved in upstream code (this came from #8144 which this PR is based on).